chapter14/compose.yaml

```diff
@@ -0,0 +1,335 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can't see in the github review UI if this is a symbolic link to the official compose in the root directory. If not please make sure it is.